source/octf/analytics/statistics/Distribution.h

```diff
@@ -28,25 +28,23 @@ class Distribution {
      * The buckets ranges are specified in constructor.
      *
      * An example of histogram for following buckets configuration:
-     * bucketSize = 10
-     * bucketPower = 2
+     * ranges = 10
+     * base = 2
      *
      * +--------------+------------+------------+-----+------------+
      * | Bucket level |  Range 0   |  Range 1   | ... | Range 9    |
      * +--------------+------------+------------+-----+------------+
-     * |            0 | [0..1]     | [2..3]     | ... | [8..9]     |
+     * |            0 | [0..0]     | [1..1]     | ... | [9..9]     |
```

Comment:
It seems there are 10 buckets in level 0, and 9 buckets in all other levels.
This also means that parameter "ranges" has a not precise description.

I suggest reverting to "Last range" in last column title and maybe also changing "Number of ranges in bucket" to "Max number of ranges in bucket" or something like that to keep documentation valid

Reply:
Row describes entire bucket. Level 0 reflects to bucket 0 which consists of 10 ranges. Level 1 (bucket 1) has 10 ranges as well. I think it's consistent.